pkg/apis/stork/register.go

```diff
@@ -2,7 +2,7 @@ package stork
 
 const (
 	// GroupName is the group name of the CRD
-	GroupName = "stork.libopenstorage.com"
+	GroupName = "stork.openstorage.org"
```

Comment:
I think we should leave that as libopenstorage.org which is the GitHub org name for the project.